go.mod

```diff
@@ -61,6 +61,7 @@ require (
 	golang.org/x/term v0.31.0
 	google.golang.org/api v0.220.0
 	google.golang.org/genproto v0.0.0-20250207221924-e9438ea467c6
+	gopkg.in/AlecAivazis/survey.v1 v1.8.8
```

Comment:
This package has been deprecated, we should not be extending use of it https://github.com/AlecAivazis/survey/tree/v1.8.8